File: PRAK305/PRAK305-2210817310015-RIDHANISETIADI.c
#include<stdio.h>
int main()
{
    int A,hari,jam,menit,detik;
    printf("Masukkan detik : ");
    scanf("%d",&A);
    hari=A/86400;
    jam=(A/3600)%24;
    menit=(A/60)%60;
    detik=A%60;
if(A<=3600){
    printf("%.2d:%.2d:%.2d\n\n",jam,menit,detik);
}else if(A>3600 && A<86400){
    printf("%.2d:%.2d:%.2d\n\n",jam,menit,detik);
}else if (A>=86400){
    printf("%d hari %.2d:%.2d:%.2d\n\n",hari,jam,menit,detik);
}
return 0;
}

File: PRAK305/PRAK305-2210817310015-RIDHANISETIADI.py
detik=int(input("Masukkan jumlah detik :"))
hari=int(detik // 86400)
jam=int(detik/3600)% 24
menit=int(detik/60)%60
d=int(detik%60)
if detik <= 3600 :
    print("%.2d:%.2d:%.2d"%(jam,menit,d))
elif(detik>3600 and detik<86400):
    print("%.2d:%.2d:%.2d\n"%(jam,menit,d))
elif(detik>=86400):
    print("%d hari %.2d:%.2d:%.2d\n"%(hari,jam,menit,d))
    




